src/Resources/contao/classes/FrontendTemplate.php

```diff
@@ -240,103 +239,6 @@ public function hasAuthenticatedBackendUser()
 
 
 	/**
-	 * Add the template output to the cache and add the cache headers
-	 */
-	protected function addToCache()
```

Comment:
> You could only modify the cache key which is now simply irrelevant :)

I could have done anything I want in there up to telling my coffee machine to brew a fresh cup, but as stated above, this is arguable and I am willing to sacrifice the hook.